sdk/nodejs/src/eventQueueAS.ts

```diff
@@ -0,0 +1,129 @@
+import { DVCEvent } from './types'
+import { DVCRequestEvent } from './models/requestEvent'
+import { DVCPopulatedUser } from './models/populatedUser'
+import { BucketedUserConfig, DVCLogger } from '@devcycle/types'
+
+import { getBucketingLib } from './bucketing'
+import { EventQueueInterface, EventTypes } from './eventQueue'
+import { publishEvents } from './request'
+
+type UserEventsBatchRecord = {
+    user: DVCPopulatedUser,
+    events: DVCRequestEvent[]
+}
+export type FlushPayload = {
+    payloadId: string,
+    eventCount: number,
+    records: UserEventsBatchRecord[]
+}
+
+type options = {
+    flushEventsMS?: number,
```

Comment:
this param name wasn't obvious that it's the interval, what about something like `eventFlushInterval`?